android/expoview/src/main/java/host/exp/exponent/utils/ScopedPermissionsRequester.kt

```diff
@@ -133,6 +133,7 @@ class ScopedPermissionsRequester(private val experienceKey: ExperienceKey) {
       Manifest.permission.WRITE_CONTACTS -> R.string.perm_contacts_write
       Manifest.permission.READ_EXTERNAL_STORAGE -> R.string.perm_camera_roll_read
       Manifest.permission.WRITE_EXTERNAL_STORAGE -> R.string.perm_camera_roll_write
+      Manifest.permission.ACCESS_MEDIA_LOCATION -> R.string.perm_access_media_location
```

Comment:
I think it has to be wrapped in the build version check 🤔

Reply:
It's just a string under the hood, so if it compiles for us (compileSdkVersion=30) then it should work everywhere. On older devices, this case will just never be resolved, I think.

Reply:
ok, you can merge it ;)